internal/types/action/action.go

```diff
@@ -8,42 +8,45 @@ type Type uint
 
 // not using iota intentionally, since the values are stored in the db as well.
 const (
-	Unknown          Type = 0
-	List             Type = 1
-	Create           Type = 2
-	Update           Type = 3
-	Read             Type = 4
-	Delete           Type = 5
-	Authenticate     Type = 6
-	All              Type = 7
-	AuthorizeSession Type = 8
-	AddGrants        Type = 9
-	RemoveGrants     Type = 10
-	SetGrants        Type = 11
-	AddPrincipals    Type = 12
-	SetPrincipals    Type = 13
-	RemovePrincipals Type = 14
-	Deauthenticate   Type = 15
-	AddMembers       Type = 16
-	SetMembers       Type = 17
-	RemoveMembers    Type = 18
-	SetPassword      Type = 19
-	ChangePassword   Type = 20
-	AddHosts         Type = 21
-	SetHosts         Type = 22
-	RemoveHosts      Type = 23
-	AddHostSets      Type = 24
-	SetHostSets      Type = 25
-	RemoveHostSets   Type = 26
-	Cancel           Type = 27
-	AddAccounts      Type = 28
-	SetAccounts      Type = 29
-	RemoveAccounts   Type = 30
-	ReadSelf         Type = 31
-	CancelSelf       Type = 32
-	ChangeState      Type = 33
-	DeleteSelf       Type = 34
-	NoOp             Type = 35
+	Unknown                   Type = 0
```

Comment:
Same comment re using `iota`

Reply:
I followed the comment above this const block which said they are not using iota intentially since the values are stored in the db.

Reply:
I guess its easier to see a change...